app/assets/stylesheets/footer.scss

```diff
@@ -0,0 +1,55 @@
+// footer
+
+.footer {
+  background-color: #48a610;
+  height: 100px;
+  text-align: center;
+  padding-top: 25px;
+  font-size: 12px;
+  font-family: 'Questrial', sans-serif;
+  width: 100vw;
+  z-index: 10;
+}
+
+.footer i {
+  color: #48a610;
+  background-color: white;
+  padding: 8px;
+  border-radius: 5px;
+  margin:	 0px 5px 2px 5px;
+}
+
+#copyright-icon {
+  color: white;
+  background-color: #48a610;
+  font-size: 12px;
+  padding: 0px;
+}
+
+.footer-text  {
+  color: white;
+  font-size: 12px;
+}
+.footer-text-underline p {
+  display: inline-block;
+  color: white;
+  padding-top: 10px;
+  line-height: 0%;
+  font-size: 12px;
```

Comment:
Not necessarily something you need to do now, but in general whenever you see repeated code, you should try to refactor the code to remove the duplication. ie. 'DRY'-up your code. 
In this case I would probably set a 12px default font size on the entire site and overwrite it wherever necessary.